code/game/cargo_forwarding.dm

```diff
@@ -284,7 +284,25 @@
 	else
 		initialised_type = get_weighted_order()
 	initialised_order = new initialised_type
-	containertype = initialised_order.must_be_in_crate ? /obj/structure/closet/crate : /obj/structure/largecrate
+	//Sadly cannot use switch here
+	if(initialised_order.must_be_in_crate)
+		if(istype(initialised_order,/datum/centcomm_order/department/engineering))
+			containertype = /obj/structure/closet/crate/secure/engisec
+			access = list(access_engine_minor)
+		else if(istype(initialised_order,/datum/centcomm_order/department/medical))
+			containertype = /obj/structure/closet/crate/secure/medsec
+			access = list(access_medical)
+		else if(istype(initialised_order,/datum/centcomm_order/department/science))
+			containertype = /obj/structure/closet/crate/secure/scisec
+			access = list(access_science)
+		else
+			containertype = /obj/structure/closet/crate
+	else
+		containertype = /obj/structure/largecrate
+	var/max_capacity = INFINITY // large crates can hold anything on creation i'm pretty sure
```

Comment:
This might have unintended consequences. Is there a specific reason it needs to be insanely high (more than like 100)?

Reply:
the default storage_capacity of all normal openable crates which use the var is 50 and totally unchanged in code, INFINITY is just a signal to ignore the check later for largecrates

Reply:
It would help if my brain saw var/ instead of thinking it was a part of the crate for some reason. We good.